doc/user-guide/github.md

```diff
@@ -125,3 +125,53 @@ If workflow needs an update a relevant message will be displayed:
 
 Please review the workflow file and update if needed.
 
+## Secrets
+
+To avoid uploading an `.env` file to Github, you may define them in your `ploomber-cloud.json` and Ploomber Cloud will read them from your environment variables. Here are the steps:
+
+1. Set secrets as environment variables using `export key=value`
```

Comment:
this is a bit confusing.

if you use github secrets, then, you never execute `export key=value`, because, this:

```
    env:
      key1: ${{ secrets.key1 }}
```

sets the environment variable

I think we can remove the whole `export key=value` references here